src/core/config.js

```diff
@@ -35,6 +35,7 @@ export default function(vm) {
       crossOriginLinks: [],
       relativePath: false,
       topMargin: 0,
+      loadNavbarOnCover: true,
```

Comment:
how about default `false` ?

WDYT ?

Reply:
We are displaying by default now, so I think it’s better to be consistent.

Reply:
makes sense